src/main/java/wolox/training/models/Book.java

```diff
@@ -124,63 +126,70 @@ public String getAuthor() {
     }
 
     public void setAuthor(String author) {
-        this.author = author;
+        this.author = Preconditions.checkNotNull(author,"Author must not be null");
```

Comment:
Remember that the String should not be empty either. There's plenty of validations you could make (for instance, that the string doesn't contain numbers or other invalid characters) but for now we're gonna stick to this 😅